woocommerce/woocommerce-gutenberg-products-block
# Product Gallery block: Disable zoom when hovering the Next/Previous buttons

## What
This PR disables the zoom when hovering over the Next/Previous buttons.

Fixes #11504 

## Why
Since the user has an intention to replace the image, we should not keep the image zooming while they move the mouse over these buttons.



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. From your WordPress dashboard, go to Appearance > Themes. Make sure you have a block-based theme installed and activated. If not, you can install one from the Add New option. Block-based themes include "Twenty-twenty Three," "Twenty-twenty Four," etc.
2. On the left-hand side menu, click on Appearance > Editor > Templates
3. Find and select the 'Single Product' template from the list.
4. When the Classic Product Template renders, click on Transform into Blocks. This will transform the Classic template in a block template if you haven't done it before.
5. Inside the Page editor, click on the '+' button to add a new block.
6. In the block library that pops up, search for the 'Product Gallery' block. Click on it to add the block to the template.
7. Click on Save;
8. Visit a product page of a Variable Product;
9. Move the mouse over the large image and make sure the zoom is enabled. Move the mouse over the previous or next buttons and make sure the image is resized to its original and the zoom is not being applied to the image while the mouse is one of the buttons.


## Screenshots or screencast



**Before**


https://github.com/woocommerce/woocommerce-blocks/assets/20469356/da5a2612-45b5-449f-9fdb-49cfc6e946da



**After**


https://github.com/woocommerce/woocommerce-blocks/assets/20469356/bac8f49b-e1ca-4bae-9276-7495479d4883



## WooCommerce Visibility


Required:
